pom.xml

```diff
@@ -122,31 +127,6 @@
                     <failOnMissingWebXml>false</failOnMissingWebXml>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
Could you elaborate on the removal of this plugin, please?

Reply:
Fixed in d3cd3f4c88c67b3621b6714e4af99b3048ad605d